indico/modules/events/requests/templates/event_request_details.html

```diff
@@ -5,10 +5,19 @@
 
 {% set has_req = req and not req.state.name == 'withdrawn' %}
 {% set show_management = is_manager and has_req %}
+{% set button_title -%}
+    {%- if not req -%}
+        {% trans %} Send request {% endtrans %}
```

Comment:
Actually, we strip this whitespace when extracting it, so it makes no real difference but it's confusing to have this whitespace in the template